package.json

```diff
@@ -24,21 +24,18 @@
     "index.js"
   ],
   "devDependencies": {
-    "babel": "^6.1.18",
-    "babel-preset-es2015": "^6.1.18",
-    "babel-tape-runner": "^2.0.0",
+    "ava": "^0.11.0",
```

Comment:
The AppVeyor 5.x build failed again, I've restarted some builds and jobs as comparing AVA 0.11.0 AppVeyor builds I've a suspicion this could actually be an AppVeyor issue rather than ours!